client/tailwind.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  purge: ["./src/**/*.{js,jsx,ts,tsx}", "./public/index.html"],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    extend: {
+      colors: {
+      primaryPink: "#f4eeff", // main bg
+      secondaryPurple: "#a6b1e1", // secondary bg
+      lightPurple: "#dcd6f7", // dropdown menu, tech stack tags
+      darkPurple: "#424874", // headers, top titles etc
+      veryDarkPurple: "#2f2e41", // some titles
+      pinkishGray: "#d0cde1", // icons
+      ourGray: "#c4c4c4", // no img
```

Comment:
This is my suggestion for name of colour: 
```js
primary-bg-color: "#f4eeff", // main bg & senpai page bg
secondary-bg-color: "#a6b1e1", // kohai page bg
tertiary-bg-color: "#dcd6f7", // dropdown menu, tech stack tags
primary-text-color: "#424874", // headers, top titles etc
```

I couldn't find `veryDarkPurple` on our mockup on Figma. Maybe that's not the colour that I intended to set.

Also, `pinkishGray` for some icons are not needed for now. As long as it's kind of shaded colour that's fine.
`ourGray`that you might find on the dummy profile picture is not needed too. We wanna use some default pictures for it.

@miyabitanimchi Hope it makes sense😄